src/js/node/http.ts

```diff
@@ -1060,246 +1072,247 @@ function onServerResponseClose() {
 }
 
 let OriginalWriteHeadFn, OriginalImplicitHeadFn;
-class ServerResponse extends OutgoingMessage {
-  declare _writableState: any;
+const privateAttribute = Symbol("privateAttribute");
+function getPrivate(obj, key) {
```

Comment:
instead of this, can you (unfortunately) use a separate symbol for each private property. this way we can avoid the extra object

maybe its worth looking into `$putByIdDirectPrivate` also. not sure if it would be meaningful to use it over symbols.

Reply:
I'm getting an error when using $putByIdDirectPrivate and $getByIdDirectPrivate
```
[RequestContext] create (src.bun.js.api.server.NewRequestContext(false,true,src.bun.js.api.server.NewServer(ZigGeneratedClasses.JSDebugHTTPServer,false,true))@20000610100)
ASSERTION FAILED: symbol
/Users/jarred/actions-runner/_work/WebKit/WebKit/Source/JavaScriptCore/bytecompiler/NodesCodegen.cpp(1726) : RegisterID *JSC::BytecodeIntrinsicNode::emit_intrinsic_putByIdDirectPrivate(BytecodeGenerator &, RegisterID *)
1   0x10695377c WTFCrash
2   0x106390220 Bun::Process_functionDlopen(JSC::JSGlobalObject*, JSC::CallFrame*)
3   0x106e5e01c JSC::BytecodeIntrinsicNode::emit_intrinsic_putByIdDirectPrivate(JSC::BytecodeGenerator&, JSC::RegisterID*)
4   0x106e79660 JSC::ExprStatementNode::emitBytecode(JSC::BytecodeGenerator&, JSC::RegisterID*)
5   0x106e79480 JSC::SourceElements::emitBytecode(JSC::BytecodeGenerator&, JSC::RegisterID*)
6   0x106e793d0 JSC::BlockNode::emitBytecode(JSC::BytecodeGenerator&, JSC::RegisterID*)
7   0x106e79480 JSC::SourceElements::emitBytecode(JSC::BytecodeGenerator&, JSC::RegisterID*)
8   0x106e80254 JSC::FunctionNode::emitBytecode(JSC::BytecodeGenerator&, JSC::RegisterID*)
9   0x106e205f8 JSC::BytecodeGenerator::generate(unsigned int&)
```